packages/logs/src/boot/startLogs.ts

```diff
@@ -88,6 +91,16 @@ export function doStartLogs(
   }
   errorObservable.subscribe(reportError)
 
+  if (isExperimentalFeatureEnabled('forward-reports')) {
+    trackReports(['deprecation', 'intervention'], (message: string, report: Report) => {
+      const loggerMethod = report.type === 'deprecation' ? 'warn' : 'error'
+      errorLogger[loggerMethod](message, { report })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
      const status = report.type === 'deprecation' ? StatusType.warn : StatusType.error
      errorLogger.log(message, { report }, status)
```